docker/ubuntu/baseimage-20.04/Dockerfile

```diff
@@ -6,6 +6,10 @@ FROM $REGISTRY/ubuntu:focal AS build
 # BOOTSTRAP {{{
 
 ARG DEBIAN_FRONTEND=noninteractive
+RUN tee /etc/apt/apt.conf.d/bare_installs.conf <<EOF
+APT::Install-Suggests "0";
+APT::Install-Recommends "0";
+EOF
```

Comment:
I find this reduces clarity, but I take it you're of a different opinion.

Reply:
I got tired of typing it every time I was running the image…

Reply:
Maybe just do both?

Reply:
That does not make sense to me.

I already use something like `apt() { sudo apt "$@"; }` when running an image, and it looks like I can change it to `apt() { sudo apt --no-install-recommends "$@"; }` and still run `apt update`, `apt remove`, etc…

So I'll remove the config.